app/models/manageiq/providers/google/cloud_manager/provision/cloning.rb

```diff
@@ -1,6 +1,6 @@
 module ManageIQ::Providers::Google::CloudManager::Provision::Cloning
   def do_clone_task_check(clone_task_ref)
-    source.with_provider_connection do |google|
+    source.with_provider_connection(:service => 'compute') do |google|
```

Comment:
There is another `source.with_provider_connection` here https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/google/cloud_manager/provision/cloning.rb#L46